model_utils/tests/tests.py

```diff
@@ -868,6 +867,8 @@ def test_select_subclass_just_self(self):
         ])
 
 
+    @skipIf(
```

Comment:
Hmm, first thing in this PR I'm not quite happy with. Did you try adjusting this test to use some other unrelated model from this app's test models, rather than using `auth.User`?